scapy/contrib/gtp_v2.py

```diff
@@ -262,14 +265,18 @@ def answers(self, other):
                 self.payload.answers(other.payload))
 
 
-class IE_IPv4(gtp.IE_Base):
-    name = "IE IPv4"
+class IE_IP_Address(gtp.IE_Base):
```

Comment:
This is an unnecessary rename to me. Why did you do that?

Reply:
Earlier it was only IPv4
But now, this IE represents both IPv4 & IPv6 - hence to correctly identify its field, I renamed it.